### ISSUE TITLE
Criterias to grade each proposal

### ISSUE THREAD

**Issue by slorber** [slorber/scalable-frontend-with-elm-or-redux#10]
Hey,

There are already a bunch of interesting proposals with pros and cons, and people coming here may not really know which one to take a look at.

I'd like to have some criterias to grade each submission, so that we know where one proposal shine more than others. What do you think?

I'm thinking of:
- How decoupled are the components
- Can the work be easily be split across different teams
- How much boilerplate does it require
- How easy it is to complicate the counter business rule or move a dom component in a very different place in the DOM

Any other things you see?

Also can we grade each proposal objectively or should contributors grade each others proposals?

**Comment by ghost** [comment]
If I had a vote to give weights to these criteria, "how decoupled are components" would be #/ 1. Changes in a front end client happen so frequently that I find the most beneficial "thing" a framework can offer is extremely isolated components. The ability to rip out a module, replace it with an entirely new design, and re-wire the existing public interface is paramount. If changing one module requires anything but the most trivial changes to some other portion of the app, your design is failing, and I don't think any other criteria really matter at that point.

**Comment by slorber** [comment]
I have to agree with that @aft-luke :) The component implementation details should not leak into other components.
